modules/lightmapper_rd/lightmapper_rd.cpp

```diff
@@ -428,6 +428,7 @@ void LightmapperRD::_create_acceleration_structures(RenderingDevice *rd, Size2i
 					SWAP(edge.a, edge.b);
 					SWAP(edge.na, edge.nb);
 					SWAP(uv2.a, uv2.b);
+					SWAP(uv2.indices.x, uv2.indices.y);
```

Comment:
A wise man once told me:

> [wouldn't be surprised if it's a one line fix tbh](https://twitter.com/passivestar_/status/1790018831018021348)

Reply:
_Narrator:_ "It was."

Reply:
Narrator: "And just like that, the issue was resolved. A single line of code, the smallest of changes, had made all the difference."